src/styles/styles.js

```diff
@@ -16,6 +16,8 @@ import positioning from './utilities/positioning';
 import codeStyles from './codeStyles';
 import visibility from './utilities/visibility';
 import optionAlternateTextPlatformStyles from './optionAlternateTextPlatformStyles';
+import pointerEventsNone from './pointerEventsNone';
+import overflowXHidden from './overflowXHidden';
```

Comment:
P.S. adding this stuff just because I noticed it was throwing warnings on iOS. 
It doesn't _really_ belong in this PR, but shouldn't hurt anything.